packages/gitbook-v2/src/lib/data/api.ts

```diff
@@ -206,8 +206,8 @@ export function createDataFetcher(
 
 const getUserById = cache(async (input: DataFetcherInput, params: { userId: string }) => {
     'use cache';
-    return trace(`getUserById(${params.userId})`, async () => {
-        return wrapDataFetcherError(async () => {
+    return wrapCacheDataFetcherError(async () => {
+        return trace(`getUserById(${params.userId})`, async () => {
```

Comment:
It swaps both so that we can see the message of the trace as "failed in" instead of succeeded